src/activation.jl

```diff
@@ -118,7 +119,7 @@ elu(x::RealOrFloatType, α = one(x)) = vifelse(x ≥ 0, x / one(x), α * (exp(x)
 activation function.
 """
 function gelu(x::RealOrFloatType)
-    p = oftype(x / 1, π)
+    p = oftype(x / 1, Float64(π))
```

Comment:
Why is this hardcoding the type here?

Reply:
Zygote willl fail for `oftype(..., ::Irrational)`